app/liquid_tags/jsitor_tag.rb

```diff
@@ -21,18 +22,21 @@ def render(_context)
 
   def jsitor_link_parser(link)
     parsed_link = ActionController::Base.helpers.strip_tags(link.strip).gsub("amp;", "")
-
-    return parsed_link if link_valid?(parsed_link)
-
-    jsitor_error
+    link_valid?(parsed_link)
   end
 
   def link_valid?(link)
-    URL_REGEXP.match link
+    return link if URL_REGEXP.match link
+    return "https://jsitor.com/embed/#{link}" if ID_REGEXP.match link
+
+    jsitor_error
   end
 
   def jsitor_error
-    raise StandardError, "Invalid JSitor link. Link should have /embed/. ex: https://jsitor.com/embed/1QgJVmCam. Parameters are optional"
+    raise StandardError, "Invalid JSitor link. Link should have /embed/.
```

Comment:
I don't think we need to include the whole explanation. 

Maybe we can just say: `Invalid JSitor link. Please read the editor guide for more information`

Reply:
I knew it. I was thinking to add a link to the guide on error too. haha. ill change this one.